README.md

```diff
@@ -21,30 +36,102 @@ from fastapi_nextauth_jwt import NextAuthJWT
 app = FastAPI()
 
 JWT = NextAuthJWT(
-    secret="y0uR_SuP3r_s3cr37_$3cr3t",
+    secret="y0uR_SuP3r_s3cr37_$3cr3t", # Leave this out to automatically read the NEXTAUTH_SECRET env var
 )
 
 @app.get("/")
 async def return_jwt(jwt: Annotated[dict, Depends(JWT)]):
     return jwt
 ```
 
-There are a few configuration options available in the NextAuthJWT constructor, but the most important one is `secret`,
-which should be equivalent to `NEXTAUTH_SECRET` on the Next.js side. Please don't hardcode the secret like this in a real application,
-but rely on the `NEXTAUTH_SECRET` env var :)
+## Configuration Options
+
+### Essential Settings
+
+- **secret** (str): The secret key used for JWT operations. Should match `NEXTAUTH_SECRET` in your Next.js app.
+  ```python
+  JWT = NextAuthJWT(secret=os.getenv("NEXTAUTH_SECRET"))
+  ```
```

Comment:
Since this is read from NEXTAUTH_SECRET by default, maybe this is more clear
```suggestion
- **secret** (str): The secret key used for JWT operations. Should match `NEXTAUTH_SECRET` in your Next.js app. Leave this out to automatically read the `NEXTAUTH_SECRET` environment variable.
  ```python
  JWT = NextAuthJWT(secret=os.getenv("YOUR_SECRET_ENV_VAR_NAME")))
  ```
```